engine/src/main/java/org/camunda/bpm/engine/impl/util/CallableElementUtil.java

```diff
@@ -40,9 +47,38 @@ public static ProcessDefinitionImpl getProcessDefinitionToCall(VariableScope exe
     String processDefinitionKey = callableElement.getDefinitionKey(execution);
     String tenantId = callableElement.getDefinitionTenantId(execution);
 
+    return getCalledProcessDefinition(execution, callableElement, processDefinitionKey, tenantId);
+  }
+
+  public static ProcessDefinition getStaticallyBoundProcessDefinition(BaseCallableElement callableElement, String tenantId) {
+    if(callableElement.hasDynamicReferences()){
+      return null;
+    }
+
+    VariableScope emptyVariableScope = new StartProcessVariableScope();
+
+    if (callableElement.getTenantIdProvider() instanceof ElValueProvider) {
+      tenantId = callableElement.getDefinitionTenantId(emptyVariableScope);
+    }
+
+    String processDefinitionKey = callableElement.getDefinitionKey(emptyVariableScope);
+    try {
+      return getCalledProcessDefinition(emptyVariableScope, callableElement, processDefinitionKey, tenantId);
+    } catch (ProcessEngineException e) {
+      UTIL_LOGGER.debugCouldNotResolveCallableElement(e);
```

Comment:
Let's try to provide as much information as we can to the users like the callable element or the definition key and tenant. That way the exception message should be a bit more useful.

Reply:
The info should already be part of the exception we get in almost all cases. Adding it again would be quite redundant.

Reply:
You have a better overview of what the exceptions look like when something cannot be resolved here. If you think all the information necessary for a user to identify the root cause of the exception is present, that's fine with me. 🙂 I was just wondering if we should add more information as to what could not be resolved here. 👍

Reply:
We could log the call activity id and/or the calling process id. Though we would have to pass them to the method just for that, which is fine I guess. We could also pass up the exception. Though I kind of like the semantics that it returns null if no fitting process can be found.

Reply:
I would also keep the semantics, let's not pass up the exception. Maybe it already is a fair compromise to use the process definition key of the definition to resolve in the exception message. We can start with that as this is all the information that's easily accessible at the moment. If there is a need for more information, we can still add it later on. What do you think?